### ISSUE TITLE
Missing documentation

### ISSUE THREAD

**Issue by azariz** [prismicio/python-kit#47]
No documentation found for the Python kit on the developers page as stated in the readme.

> get a thorough introduction of how to use prismic.io kits, including this one.

And the package documentation is also incomplete, not all functions are described.

**Comment by lietu** [comment]
Indeed, the links point to some generic site where the word `Python` is not mentioned even once.

And to make this worse nothing seems to actually work when you follow instructions - the prismic.io website says to use `/api/v2` endpoint (doesn't list Python as an option at all), but apparently it should be `/api`.

The commands can throw really raw errors from HTTP responses but doesn't then even give any idea what the request it actually made was.

The functions themselves don't even have decent documentation that the `help()` builtin could show, but I end up having to guess what they might want as arguments.

When they do work the function return values are .. well .. less than optimal and quite surprising. For some reason `p.query` returns an `OrderedDict`, as if it mattered if `page` is 1st and `results_per_page` 2nd in that iterable. I would understand the use of `OrderedDict` for just the `results`, but this is `OrderedDict` in `OrderedDict`  in `OrderedDict`  in `OrderedDict` that makes the output on the Python shell really damn hard to read when trying to comprehend the structure.

Somehow those responses are also so broken that `dict(p.query(...))` ends up raising

```
TypeError: attribute of type 'NoneType' is not callable
```

I gotta say this is one of the worst libraries I've seen out there, and I think I'll be much better off using the Rest API directly as this does nothing but give me headaches.
